code/src/components/product-skeleton/ProductSkeleton.tsx

```diff
@@ -16,7 +16,7 @@ const ProductSkeleton = () => {
         width={254}
         animation="pulse"
       />
-      <AppBox className="spa-product-skeleton__footer">
+      <AppBox className="spa-product-skeleton__footer" width={234}>
```

Comment:
You should move this property to SCSS. Also, is it flexible with this hardcoded width on different screen sizes? Please provide a video in the description

Reply:
Yes it works as intended. The skeletons themselves have hardcoded values, so on different screen sizes, the container for the footer skeletons stretch out. having fixed width for the container fixes the issue. Moving this property to the SCSS, doesn't seem to work. It gets overwritten somehow.

Reply:
Then try to use `!important`. We used it several times because styles of mui is a bit hard to override in terms of specifficity

Reply:
Done